AppServer_Java/src/com/google/appengine/api/taskqueue/dev/AppScaleTaskQueueClient.java

```diff
@@ -101,6 +99,24 @@ public AppScaleTaskQueueClient() {
         return leaseResponse;
     }
 
+    public TaskQueuePb.TaskQueueDeleteResponse delete(TaskQueuePb.TaskQueueDeleteRequest deleteRequest) {
+        deleteRequest.setAppId(getAppId());
+        Request request = new Request();
+        request.setMethod("Delete");
+        request.setServiceName(SERVICE_NAME);
+        request.setRequestAsBytes(deleteRequest.toByteArray());
+        Response response = sendRequest(request);
+        if (response.hasApplicationError()) {
+            throw new ApiProxy.ApplicationException(
+                response.getApplicationError().getCode(),
+                "TaskQueue Delete operation failed"
```

Comment:
I couldn't find a way to get that with com.google.apphosting.utils.remoteapi.RemoteApiPb.Response.

We'd need to swap out the request exchange mechanism between the Java AppSever and the TaskQueue server to do that efficiently, which imho falls out of scope for this branch.